controllers/scopeinstance_controller.go

```diff
@@ -114,7 +242,40 @@ func (r *ScopeInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *ScopeInstanceReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	enqueueOperator := handler.EnqueueRequestsFromMapFunc(r.mapToScopInstace)
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&operatorsv1.ScopeInstance{}).
+		Watches(&source.Kind{Type: &operatorsv1.ScopeTemplate{}}, enqueueOperator).
 		Complete(r)
 }
+
+func (r *ScopeInstanceReconciler) mapToScopInstace(obj client.Object) (requests []reconcile.Request) {
+
+	if obj == nil || obj.GetName() == "" {
+		return
+	}
+
+	// Requeue all Scope Instance in the resource namespace
+	ctx := context.TODO()
+	scopeInstanceList := &operatorsv1.ScopeInstanceList{}
+
+	log.Log.Info("getting object name", "object", obj.GetName())
+	if err := r.Client.List(ctx, scopeInstanceList); err != nil {
+		//&client.ListOptions{FieldSelector: fields.ParseSelectorOrDie(fmt.Sprintf("spec.ScopeTemplateName=%v", obj.GetName()))}); err != nil {
+		r.log.Error(err, "error listing scope instances")
```

Comment:
return an empty array if you hit an error, no need to log.

Reply:
returned nil